src/plugins/intel_cpu/src/nodes/kernels/rdft_kernel.hpp

```diff
@@ -60,7 +60,15 @@ struct jit_dft_kernel_f32 : public jit_dft_kernel, public jit_generator {
     public:
         DECLARE_CPU_JIT_AUX_FUNCTIONS(jit_dft_kernel_f32)
 
-        jit_dft_kernel_f32(bool is_inverse, enum dft_type type) : jit_dft_kernel(is_inverse, type), jit_generator(jit_name()) {}
+        jit_dft_kernel_f32(bool is_inverse, enum dft_type type) : jit_dft_kernel(is_inverse, type), jit_generator(jit_name()) {
+            int simd_size = vlen / sizeof(float);
```

Comment:
```suggestion
            int simd_size = vlen / type_size;
```
or if `vlen` and `type_size` can be `constexpr`
```suggestion
            constexpr int simd_size = vlen / sizeof(float);
```
?
Consider add before loop:
```cpp
perm_low_values.reserve(simd_size);
perm_high_values.reserve(simd_size);
```